packages/storage-adapters/resolve-storage-mongo/test/index.tests.js

```diff
@@ -5,15 +5,15 @@ import { ConcurrentError } from 'resolve-storage-base'
 
 jest.mock('mongodb', () => {
   const sinon = require('sinon')
-  let foundArray = []
+  let foundCursor = { current: 0, items: [] }
   let isRejectInsert = false
 
   return {
     _setFindResult: array => {
       if (Array.isArray(array)) {
-        foundArray = array
+        foundCursor = { current: 0, items: array }
```

Comment:
foundCursor = { current: 0, items: Array.isArray(array) ? array : [] }

Reply:
Check two times to make sure?